src/typeDefs/Order.ts

```diff
@@ -0,0 +1,78 @@
+import { gql } from "graphql-tag";
+import { OrderStatus } from "@prisma/client";
+
+const typeDefs = gql`
+  enum OrderStatus {
+    PENDING
+    COMPLETED
+  }
+
+  type Order {
+    orderId: ID!
+    customerName: String!
+    customerEmail: String!
+    customerPhoneNum: String!
+    billingAddressLine1: String!
+    billingAddressLine2: String
+    billingCity: String!
+    billingZip: String!
+    billingCountry: String!
+    shippingAddressLine1: String!
+    shippingAddressLine2: String
+    shippingCity: String!
+    shippingZip: String!
+    shippingCountry: String!
+    status: OrderStatus!
+    createdAt: String!  # Consider using DateTime if supported
+  }
+
+  type Query {
+    getOrder(id: ID!): Order
+    listOrders: [Order]!
+  }
+
+  type Mutation {
+    createOrder(input: CreateOrderInput!): Order
+    updateOrderStatus(id: ID!, status: OrderStatus!): Order
```

Comment:
Add deletion here, since it needs to be defined in your typeDefs and not just in your resolvers.